src/logicsig.ts

```diff
@@ -128,7 +128,7 @@ export class LogicSig implements LogicSigStorageStructure {
    * @param msig - Multisig account as \{version, threshold, addrs\}
    */
   sign(secretKey: Uint8Array, msig?: MultisigMetadata) {
-    if (msig === undefined) {
```

Comment:
It could be better: `if (!msig) { ... }`. `msig` will never be null, it can be `MultisigMetadata` or `undefined`.